Source/WebKit/WebProcess/Plugins/PDF/PDFPluginAnnotation.mm

```diff
@@ -93,7 +94,11 @@
 
     m_eventListener->setAnnotation(0);
 
-    m_parent->removeChild(*element());
+    m_element->document().eventLoop().queueTask(TaskSource::InternalAsyncTask, [ element = WeakPtr<Node, WeakPtrImplWithEventTargetData> { this->element() } ]() {
+        if (!element)
```

Comment:
nit: I think this would look a bit nicer:
```
if (RefPtr element = weakElement.get())
    element->remove();
```

Reply:
Ok, sure we can do that.